tests/textcase/lsp/lsp_spec.lua

```diff
@@ -1,43 +1,47 @@
 local test_helpers = require("tests.test_helpers")
 local textcase = require("textcase")
 
+-- Allow Typescript language server to start.
+-- Even though the language server is attached, it doesn't mean it is ready
+-- to receive requests. Hence, we send a request and wait for response.
+-- Then we know the language server is ready.
+local wait_for_language_server_to_start = function()
```

Comment:
Not sure if we should move this to test_helpers, because we most likely will only need it here.